iot-hub/Quickstarts/ReadD2cMessages/Program.cs

```diff
@@ -20,20 +20,20 @@ namespace ReadD2cMessages
     /// </summary>
     internal class Program
     {
-        private static Parameters _parameters;
+        private static Parameters s_parameters;
```

Comment:
move to be a method variable instead of a class field

Reply:
cool, looks like that's the case for multiple of these. I'll change it across the board